src/game/Commands/CharacterCommands.cpp

```diff
@@ -883,7 +883,7 @@ bool ChatHandler::HandleRemoveRidingCommand(char* args)
             // Journeyman Riding
             case 33391:
             {
-                if (!CharacterDatabase.PExecute("UPDATE character_skills SET value = 75, max = 75 WHERE skill = 762 AND value = 150 AND max = 150 AND guid = %u", target_guid.GetCounter()))
+                if (!CharacterDatabase.PExecute("UPDATE `character_skills` SET `value` = 75, `max` = 75 WHERE `skill` = 762 AND `value` = 150 AND `max` = `150` AND `guid` = '%u'", target_guid.GetCounter()))
```

Comment:
150 is not a column or table name

Reply:
fixed with https://github.com/vmangos/core/pull/896/commits/c552f3bdd708ebbee45b02b1a864d228c4f6dde0